test/data-ingest/mzcompose.py

```diff
@@ -36,7 +36,7 @@
         ],
     ),
     SchemaRegistry(),
-    Materialized(),
+    Materialized(ports=["16875:6875"]),
```

Comment:
Are the static ports needed for the framework to function, this was just a debugging aid for yourself?